src/main/java/org/terasology/fluid/system/FluidAuthoritySystem.java

```diff
@@ -60,6 +70,10 @@ public void fillFluidContainerItem(ActivateEvent event, EntityRef item, FluidCon
 
                         FluidUtils.setFluidForContainerItem(removedItem, fluidType);
 
+                        // Fill this fluid container up to max capacity.
```

Comment:
The real dilemma for me is that because I have extended the fluid system to accommodate some stuff around tank interactions,  I want to be calling FluidUtils directly.  If some of the logic is not in that static method,  these extensions to Fluid break.